nornir_utils/plugins/functions/print_stat.py

```diff
@@ -0,0 +1,96 @@
+import threading
+from typing import Tuple
+from nornir_utils.plugins.functions.print_result import _get_color
+
+from colorama import Fore, Style, init
+
+from nornir.core.task import AggregatedResult, MultiResult, Result
+
+
+LOCK = threading.Lock()
```

Comment:
we should use the same lock that `print_result` uses, otherwise, `print_result` and `print_stat` can interfere with each other

Reply:
imported `print_result` `LOCK` for `print_stat` function